src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -132,92 +135,82 @@ const Tabs = ({ children, id, modifyHash = true, testId, onToggle }: Props) => {
       data-testid={testId}
       ref={ref}
     >
-      <ul className="govuk-tabs__list" role="tablist">
+      <ul className="govuk-tabs__list" role={!isSingleTab ? 'tablist' : 'list'}>
         {sections.map(({ props }, index) => {
           const sectionId = props.id || `${id}-${index + 1}`;
           return (
             <li
               className={classNames('govuk-tabs__list-item', {
                 'govuk-tabs__list-item--selected': selectedTabIndex === index,
+                [styles.mobileHidden]: isSingleTab,
               })}
               key={sectionId}
-              role="presentation"
+              role={!isSingleTab ? 'presentation' : 'listitem'}
```

Comment:
What's the purpose of changing the role here? 

Also, given that this `li` element, it already has the implicit role of `listitem`. Should be able to set it to `undefined` instead